queries/thrift/highlights.scm

```diff
@@ -57,6 +60,9 @@
 ; Attributes
 
 (annotation_identifier) @attribute
+(exception_modifier) @attribute
+(field_modifier) @attribute
+(function_modifier) @attribute
```

Comment:
are modifiers annotations? or `@type.qualifier`?

Reply:
not entirely sure what `@type.qualifier` is, but they are keywords before a function like async, before a field like required, optional

Reply:
yes, then not `@attribute`. Attribute is for annotations that are not actually necessary in the code but can be added for additional meta info the compiler/refection or alike (originally for C++ attributes) or those Go serialization tags. `@type.qualifier` and `@storageclass` are typically used for stuff like `const`, `async`, `volatile` or just `@keyword`.

Reply:
gotcha, thanks! fixed now